client/src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,162 @@
+  loggedInUser: User;
+  open: boolean;
+}
+
+const Navbar = ({
+  logOut,
+  profile,
+  mySitter,
+  becomeSitter,
+  messages,
+  loggedInUser,
+  loggedIn,
+  open,
+}: Props): JSX.Element => {
+  const classes = useStyles();
+
+  const [state, setState] = useState({
+    mobileView: false,
+    drawerOpen: false,
+  });
+
+  const { mobileView, drawerOpen } = state;
+
+  useEffect(() => {
+    const setResponsiveness = () => {
+      return window.innerWidth < 1200
+        ? setState((prevState) => ({ ...prevState, mobileView: true }))
+        : setState((prevState) => ({ ...prevState, mobileView: false }));
+    };
+
+    setResponsiveness();
+
+    window.addEventListener('resize', () => setResponsiveness());
+
+    return () => {
+      window.removeEventListener('resize', () => setResponsiveness());
+    };
+  }, []);
+
+  const displayDesktop = () => {
+    return (
+      <Toolbar>
+        <Grid container>
+          <Grid item container sm={2} alignContent="center">
+            <PetsIcon fontSize="large" className={classes.icon} />
+            <Typography variant="h5" align="center" noWrap className={classes.logo}>
+              LovingSitter.
+            </Typography>
+          </Grid>
+          <Grid sm={10} item container className={classes.toolbarGrid}>
+            {getMenuButtons()}
+          </Grid>
+        </Grid>
+      </Toolbar>
+    );
+  };
+
+  const displayMobile = () => {
+    const handleDrawerOpen = () => setState((prevState) => ({ ...prevState, drawerOpen: true }));
+    const handleDrawerClose = () => setState((prevState) => ({ ...prevState, drawerOpen: false }));
+
+    return (
+      <Toolbar>
+        <Grid item container alignContent="center">
+          <PetsIcon fontSize="large" className={classes.icon} />
+          <Typography variant="h5" align="center" noWrap className={classes.logo}>
+            LovingSitter.
+          </Typography>
+        </Grid>
+
+        <IconButton
+          {...{
+            edge: 'start',
+            color: 'secondary',
+            'aria-label': 'menu',
+            'aria-haspopup': 'true',
+            onClick: handleDrawerOpen,
+          }}
+        >
+          <MenuIcon />
+        </IconButton>
+
+        <Drawer
+          {...{
+            anchor: 'right',
+            open: drawerOpen,
+            onClose: handleDrawerClose,
+          }}
+        >
+          <div className={classes.drawerContainer}>{getDrawerChoices()}</div>
+        </Drawer>
+
+        <div>{headerLogo}</div>
+      </Toolbar>
+    );
+  };
+
+  const getDrawerChoices = () => {
+    return (
+      <React.Fragment>
+        <Link to={logOut} className={classes.menuDownButton}>
+          Log out
+        </Link>
+        <Link to={profile} className={classes.menuDownButton}>
+          Go to profile
+        </Link>
+      </React.Fragment>
+    );
+  };
+
+  const headerLogo = <Typography variant="h5" component="h5"></Typography>;
+
+  const getMenuButtons = () => {
```

Comment:
This could be moved outside as well.

Reply:
created new component named GetMenuButtons